components/CollectionForm.js

```diff
@@ -44,17 +59,59 @@ const StyledSubmitMessage = styled.p`
   text-align: center;
 `;
 
+const StyledInput = styled.input`
+  font-size: 16px;
+  border-radius: 12px;
+  padding: 8px;
+  border: 1px solid var(--light-grey);
+  width: 100%;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  &:focus {
+    outline: none;
+    border: 1px solid #000;
+  }
+`;
+
 export default function CollectionForm({ onSubmitCollection, prevValues }) {
   const [confirmMessage, setConfirmMessage] = useState("");
```

Comment:
As later on you are setting `confirmMessage` to `true`, I would suggest to set the initial value to `false`.